app/src/pages/inside/testCaseLibraryPage/importTestCaseModal/messages.ts

```diff
@@ -34,4 +38,24 @@ export const messages = defineMessages({
     id: 'ImportTestCaseModal.fileSizeMessage',
     defaultMessage: 'File size should be up to {size} MB',
   },
+  specifyLocation: {
+    id: 'import.specifyLocation',
```

Comment:
```suggestion
    id: 'ImportTestCaseModal.specifyLocation',
```
And below. Could you please use the full path in the id as a key (not just createRoot but createNewRootFolder)